lms/static/js/views/image_field.js

```diff
@@ -183,13 +184,13 @@
 
             showUploadInProgressMessage: function () {
                 this.$('.u-field-upload-button').css('opacity', 1);
-                this.$('.upload-button-icon').html(this.iconProgress);
+                this.$('.upload-button-icon').html(this.iconProgressHtml.toString());
```

Comment:
Throwing out some ideas here:
```
this.$('.upload-button-icon').html(HtmlUtils.forJQuery(this.iconProgressHtml));
```
or
```
this.$('.upload-button-icon').html(HtmlUtils.for$(this.iconProgressHtml));
```
Again, these are just other alternative names for `escape()`, but maybe you'll like them more and we can have something consistent for the linter.  For example, this could still work with interpolateHtml() as such...
```
this.$('.upload-button-icon').html(HtmlUtils.for$(HtmlUtils.interpolateHtml(...)));
```